packages/rulesets/generated/spectral/az-arm.js

```diff
@@ -329,6 +329,18 @@ function isSchemaEqual(a, b) {
     }
     return false;
 }
+function createRuleFunctionWithPasses(fn) {
+    return (input, options, ctx) => {
+        const messsages = fn(input, options, ctx);
+        if (messsages.length === 0) {
+            messsages.push({
+                message: `[Verbose]this is a verbose message to indicate that this rule was passed for specific swagger schema successfully and no fix is needed, please ignore it.`,
```

Comment:
>this is a verbose message to indicate that this rule was passed for specific swagger schema successfully and no fix is needed, please ignore it. [](http://example.com/codeflow?start=35&length=144)

How about this text for the verbose message - "ARM RPC rule passed for path {path}" ?